### ISSUE TITLE
Only the Last List-Item Closes

### ISSUE THREAD

**Issue by sstoft** [evbacher/gd2md-html#15]
<li>Thing 1
<li>thing 2
<li>Thingy 3</li></ul>

Happened on both lists.

**Comment by evbacher** [comment]
What do you mean both lists?

Also, when I convert this to HTML, I get this:

```html
<h1>List testing</h1>

<ul>

<li>Thing 1
<li>thing 2
<li>Thingy 3</li></ul>
```

which is actually still valid HTML5, and it renders fine. I'm working on cleaning up some other issues with HTML list-processing, and that may clean up this inconsistency as well.

**Comment by sstoft** [comment]
I had two bullet lists, one with 3 items, and one with 2.
But had </li> only on the last item.

**Comment by evbacher** [comment]
Ah, got it. And thanks for reporting this edge case -- let me know if you notice any others.

**Comment by hayksaakian** [comment]
this is still present, and really my only complaint.

I love this addon but the list item bug is really show stopping.

**Comment by ultimike** [comment]
Agreed - I work with a lot of documents with nested lists and this bug is a showstopper for me as well.

**Comment by evbacher** [comment]
I will try to spend some time on this, though Markdown is the primary target.

**Comment by evbacher** [comment]
I believe this is fixed with recent changes. Please let us know if you see evidence of this bug still happening.
